src/views/Profile/AuthorsList/AuthorsListFilters.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Typography, Box } from '@material-ui/core';
+import i18, { langTokens } from '../../../locales/localizationInit';
+import { AdminTextField } from '../AdminTable/AdminTextField';
+import { TextFieldsEnum } from '../../../models/experts/types';
+import { AuthorListDropdown } from './AuthorListDropdown';
+import { useStyles } from './styles/AuthorsListFilters.styles';
+import { setSize, setField } from '../../../models/experts';
+import { useActions } from '../../../shared/hooks';
+
+interface IAuthorsListFiltersProps {
+  size: number;
+  author: string;
+}
+
+const optionsToShow = [10, 25, 50, 100];
+
+export const AuthorsListFilters: React.FC<IAuthorsListFiltersProps> = (
+  props,
+) => {
+  const classes = useStyles();
+  const { size, author } = props;
+  const [boundedSetSize, boundedSetField] = useActions([setSize, setField]);
+
+  const handleNotesToShowChange = (val) => {
+    boundedSetSize(val);
+  };
+
+  return (
+    <Box className={classes.filtersBox}>
+      <Typography variant="body1" className={classes.notesToShowPanel}>
+        {i18.t(langTokens.admin.showNotesAmount).split(' ')[0]}
```

Comment:
this is probably better to be split into separate `admin.showNotesAmountLeft` and `admin.showNotesAmountRight`

splitting by " " is not flexible enough, and we may want to have several words to the left. But splitting by spaces and picking just first word will not allow that

Reply:
done